### PR TITLE
Improves the use of long datasets with the Regular sequence dialog. 

### LINKED ISSUE #7388: Improve the coding for the Regular sequence dialogue
@anastasia-mbithe perhaps you may like to try this task?

Could you look at the discussion topic #7378.  I hope you will find it interesting as a topic that could perhaps be included in a maths camp? 

Within that topic I found that the regular sequence dialogue is practically unusable for long datasets.  Could you check how it is coded?  I suspect that the problem may be related to the preview, but I don't know.  Perhaps you could start by omitting the preview as a start.  Then, if it is the preview, perhaps that could have a Preview button, so it is not done by default.  

The problems may be elsewhere, instead, or as well.

### PR DESCRIPTION
**Pull request by anastasia-mbithe**
This fixes #7388 

@rdstern, I have added a checkbox that when checked, we have the preview. I created a data frame of 200,000 rows and it worked well. Using 2,000,000 rows, it is well until you check the checkbox. So, I feel for a data frame with about 2,000,000 rows, it's safe having the regular sequence without checking for the preview at the moment. 
But since some people may find the preview useful, maybe we could show the preview for the first 1000 rows. What do you think?

### CODE REVIEW

File: instat/dlgRegularSequence.vb [hunk 1]
```diff
@@ -315,4 +324,16 @@ Public Class dlgRegularSequence
...
+    Private Sub ucrChkPreview_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrChkPreview.ControlValueChanged
+        If ucrChkPreview.Checked Then
+            txtPreview.Visible = True
+            lblPreview.Visible = True
+            lblMessage.Visible = True
+        Else
+            txtPreview.Visible = False
+            lblPreview.Visible = False
+            lblMessage.Visible = False
+        End If
```
Comment (lloyddewit) [inline instat/dlgRegularSequence.vb:336]:
```suggestion
        txtPreview.Visible = ucrChkPreview.Checked
        lblPreview.Visible = ucrChkPreview.Checked
        lblMessage.Visible = ucrChkPreview.Checked
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@anastasia-mbithe code looks good, thank you
just one small suggestion

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern Please could you test? thanks

**Review by rdstern — APPROVED** [review]
@anastasia-mbithe this seems good now.

**Pull request merged**